workshops/lyon2018/intro-with-globular.ipynb

```diff
@@ -283,7 +283,7 @@
     "# We'll customize the configuration of measurement to just run a few plugins.\n",
     "# The default list of plugins is much longer (and hence slower).\n",
     "measureConfig = SingleFrameMeasurementTask.ConfigClass()\n",
-    "measureConfig.plugins.names = [\"base_SdssCentroid\", \"base_PsfFlux\"]\n",
+    "measureConfig.plugins.names = [\"base_SdssCentroid\", \"base_PsfFlux\", \"base_SkyCoord\"]\n",
```

Comment:
I'm not familiar with the measurement plugins here.  Why do we want this extra plugin in this notebook?

Reply:
I found that `base_Skycoord` was needed to populate the `coord_ra` and `coord_dec` columns. Otherwise they are all nulls or NaNs, and Firefly doesn't overlay the table.